### ISSUE TITLE
WinRM Timeout for CIS Microsoft Windows Server 2016 Benchmark v1.1.0.2 - Level 1

### ISSUE THREAD

**Issue by LennyCastaneda** [hashicorp/packer#8093]
AMI NAME:
CIS Microsoft Windows Server 2016 Benchmark v1.1.0.2 - Level 1-b1776421-c8ab-411d-8a73-34f06d3ceadf-ami-02739b39b0502b089.4

AMI ID: 
ami-00dd8c6592f567cc5

WinRM timing out after using CIS Hardened Windows 2016.

Two sets of bootstrap scripts attached below.

[bootstrap_win.txt](https://github.com/hashicorp/packer/files/3597344/bootstrap_win.txt)

[windows_auto_generated_winrm_pass.txt](https://github.com/hashicorp/packer/files/3597345/windows_auto_generated_winrm_pass.txt)

{
    "builders": [{
        "type": "amazon-ebs",
        "region": "us-west-2",
        "source_ami_filter": {
            "filters": {
                "virtualization-type": "hvm",
                "name": "CIS Microsoft Windows Server 2016 Benchmark v1.1.0.*Level 1*",
                "root-device-type": "ebs"
            },
            "owners": ["679593333241"],
            "most_recent": true
        },
        "ami_name": "TeamCityAgent.v4.0.0-{{timestamp}}",
        "ami_description": "TeamCity Build Agent image made by Packer",
        "instance_type": "m4.large",
        "iam_instance_profile": "DevOpsProvisioningInstance",
        "vpc_id": "vpc-xxxxxxxxxxx",
        "subnet_id": "subnet-xxxxxxxxxxx",
        "security_group_ids": [
            "sg-Xxxxxxxxxxxxxx"
        ],
        "user_data_file": "./windows_auto_generated_winrm_pass.txt",
        "communicator": "winrm",
        "force_deregister": true,
        "winrm_username": "Administrator",
        "winrm_insecure": true,
        "winrm_use_ssl": true,
        "winrm_use_ntlm": true,
        "tags": 
            {
                "Name"  : "TeamCityAgent.v4.0.0-{{timestamp}}",
                "Packer": "True"
            }
    }],
    "provisioners": [
        {
            "type": "powershell",
            "script": "Install-TeamCityAgentTools.ps1"
        }
    ]
}

**Comment by SwampDragons** [comment]
Related to #6951

I took at look at this and was able to get the following configs working:  https://gist.github.com/SwampDragons/a0044e88c614a507a533ee9323cf8e0c

Specifically I could only get this working when I used an autogenerated password rather than setting one in my user data file, and I had to set `"winrm_use_ntlm"=true`.  Lenny and I got a chance to hack on this at HashiConf but we couldn't get it working for him, I think because he's not behind his normal company VPN. We'll pick it back up later this week after the conference is over.
